include/amici/abstract_model.h

```diff
@@ -68,6 +68,39 @@ class AbstractModel {
                         const AmiVector &dx, int ip, const AmiVector &sx,
                         const AmiVector &sdx, AmiVector &sxdot) = 0;
 
+    /**
```

Comment:
As shortly mentioned: I'm not sure, whether implementing these additional functions into abstract_model is really what we want. I needed this, because backward steadystate integration operates on members of model (or rather: uses them, not really changes them). An alternative would be modifying the solver::setup(), in order to allow custom RHS and Jacobian functions to be passed to it.

Reply:
Good to know indeed. :+1: